packages/truffle-codec-utils/src/types/types.ts

```diff
@@ -172,6 +176,19 @@ export namespace Types {
     location?: Ast.Location;
   }
 
+  export interface OptionallyNamedType {
+    name?: string;
```

Comment:
Shouldn't this be `name: string | undefined` instead?

Reply:
In this context, I wouldn't consider that any different.